input/pagecontent/reference.md

```diff
@@ -2687,6 +2687,10 @@ To insert a parameterized rule set, use the rule set name with a list of one or
 
 As indicated, the list of values is enclosed with parentheses `()` and separated by commas `,`. If you need to put literal `)` or `,` characters inside values, escape them with a backslash: `\)` and `\,`, respectively. White space separating values is optional, and removed before the value is applied to the rule set definition.
 
+{%include tu-div.html%}
+Alternatively, a parameter value may be surrounded by double square brackets `[[` `]]`. Literal `)` and `,` characters within the double square brackets do not need to be escaped with a backslash, unless appearing after `]]` characters. Note that the full value must be surrounded with double square brackets in order for this type of processing to occur.
```

Comment:
I feel like you made it pretty clear that you have to be within the double brackets, so I took out one extra clarification to simplify it. What do you think?

```suggestion
Alternatively, a parameter value may be surrounded by double square brackets `[[` `]]`. Literal `)` and `,` characters within the double square brackets do not need to be escaped with a backslash. Note that the full value must be surrounded with double square brackets in order for this type of processing to occur.
```

Reply:
Ah, maybe my clarification needs clarification? The situation I'm attempting to describe is like this:
```
* insert MyRules([[some list [[abc]]\, etc]], "second value")
```
In the above FSH, there are two values:
- a double-bracketed `some-list [[abc]], etc`
- `"second value"`

If you don't have that escape, like this:
```
* insert MyRules([[some list [[abc]], etc]], "second value")
```
You end up with three values:
- a double-bracketed `some-list [[abc`
- `etc]]`
- `"second value"`

Reply:
In today's standup we talked about the following sentence:

> Literal `)` and `,` characters within the double square brackets do not need to be escaped with a backslash, unless appearing after `]]` characters.

@jafeltra indicated that the intent was not clear to her, so we should try to find alternative ways of saying this. Here are a few potential adjustments:

> 1. Literal `)` and `,` characters within the double square brackets do not need to be escaped with a backslash, unless appearing after `]]` characters that are intended to be a part of the parameter value.

> 2. Literal `)` and `,` characters within the double square brackets do not need to be escaped with a backslash except when part of a literal `]],` or `]])` sequence that should _not_ be interpreted as the end of the parameter.

> 3. Literal `)` and `,` characters within the double square brackets do not need to be escaped with a backslash. The only exception is when the author wants to include `]],` or `]])` as part of the parameter value. In this (very rare) case, the `)` or `,` following the `]]` should be escaped with a backslash. For example, to include `]],` as part of a parameter value within double square brackets, use `]]\,`.This needs to be done even if there is whitespace between the `]]` and `,` or `)`.

As noted in the meeting, it might also be helpful to include an example like Mint has included above.